### ISSUE TITLE
Support proseco catalogs

### ISSUE THREAD

**Issue by taldcroft** [sot/starcheck#252]
There are a number of changes needed to support proseco catalogs:

- [ ] Use Py3 and modern hopper #250.
- [ ] Define and implement changes to requirements, for example Angle Too Large (see https://github.com/sot/proseco/issues/17)
- [ ] Integration with proseco reporting (run report code and include links)

**Comment by jeanconn** [comment]
As another issue, I suspect current starcheck will give us plenty of bad pixel warnings on new catalogs that have guide stars selected that allow "faint" hot pixels on bright stars.  We'll have no way of telling which of these are valuable warnings. We probably need an independent implementation of the hot pixel check in starcheck.

**Comment by jeanconn** [comment]
Also, regard to scope, what about running starcheck 4.0 in parallel?  Too confusing?

**Comment by taldcroft** [comment]
Yes, there should be just one definitive tool for load review and approval.

> We'll have no way of telling which of these are valuable warnings.

This information should be provided in a guide selection report analogous to the acq report.  It could be a lightweight and manageable update to current starcheck to just provide links to those reports (and run the Python tools to generate the reports if the `proseco.pkl` file is available).

> We probably need an independent implementation of the hot pixel check in starcheck.

Not super keen on re-implementation.  The Python algorithm and implementation will be tested and validated, so you should be able to just call that from starcheck as needed, no?  (I looked at the code and API change is needed, but nothing that would be difficult).

**Comment by jeanconn** [comment]
> This information should be provided in a guide selection report analogous to the acq report. It could be a lightweight and manageable update to current starcheck to just provide links to those reports (and run the Python tools to generate the reports if the proseco.pkl file is available).

This still seems like information "on the side" somewhat similar to running a separate load review tool, but if you think it is much better than a separate evaluation I'll keep it in mind.

> Not super keen on re-implementation. 

I think in this case the independent implementation would actually be just running annie as part of load review... which I think you are onboard with, but probably not for the next starcheck release.

**Comment by taldcroft** [comment]
I believe we have to find acceptable workarounds and low-effort implementations on many fronts to make the deadline.  That's the driver for my suggestions.  Of course I don't *want* to see meaningless warnings and have to click through to see if it has meaning.
